thirdparty/tiny-process-library/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Third-party code should be listed in the:

https://github.com/godotengine/godot/blob/master/thirdparty/README.md
https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt